app.js

```diff
@@ -1,6 +1,7 @@
 const express = require('express');
 const cors = require('cors');
-
+const { authRouter } = require('./routes/auth');
+const cookieParser = require('cookie-parser');
```

Comment:
🚫 **[eslint]** <[import/order](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/order.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`cookie-parser` import should occur before import of `./routes/auth`
```suggestion
const cookieParser = require('cookie-parser');
const { authRouter } = require('./routes/auth');
```